pkg/trace/writer/sender.go

```diff
@@ -339,6 +352,7 @@ func (s *sender) do(req *http.Request) error {
 type payload struct {
 	body    *bytes.Buffer     // request body
 	headers map[string]string // request headers
+	retry   int32
```

Comment:
```suggestion
	retries   int32 // number of retries sending this payload
```
Think this is how you had it initially and it made more sense.